### ISSUE TITLE
gui, side bar

### ISSUE THREAD

**Issue by qinghong67** [productiveops/dokemon#28]
need option to hide the side bar. it's block of the main site.

**Comment by RicardoJeronimo** [comment]
I'm having the same issue, on mobile it's almost impossible to navigate the site.

**Comment by Sp33dFr34k** [comment]
+1 for this one, should be a fairly simple fix I'd imagine.

**Comment by jmagar** [comment]
Agreed. When viewing on a mobile device unless you request desktop site and zoom out the site is unusable

**Comment by skuduu** [comment]
Agreed, option to hide sidebar is definetly needed, or the mobile view of the GUI needs to be adjusted. Cannot view the GUI on mobile site because the sidebar is overlapping the other stuff.
